winpr/libwinpr/file/file.c

```diff
@@ -463,30 +475,32 @@ static BOOL FileUnlockFileEx(HANDLE hFile, DWORD dwReserved, DWORD nNumberOfByte
 	lock.l_len = 0;
 	lock.l_whence = SEEK_SET;
 	lock.l_type = F_UNLCK;
+
 	if (fcntl(fileno(pFile->fp), F_GETLK, &lock) == -1)
 	{
 		WLog_ERR(TAG, "F_UNLCK on %s failed with %s [0x%08X]",
-			 pFile->lpFileName, strerror(errno), errno);
+		         pFile->lpFileName, strerror(errno), errno);
 		return FALSE;
 	}
+
 #else
+
 	if (flock(fileno(pFile->fp), LOCK_UN) < 0)
 	{
 		WLog_ERR(TAG, "flock(LOCK_UN) %s failed with %s [0x%08X]",
-			 pFile->lpFileName, strerror(errno), errno);
+		         pFile->lpFileName, strerror(errno), errno);
 		return FALSE;
 	}
-#endif
 
+#endif
 	return TRUE;
 }
 
-static UINT64 FileTimeToUS(const FILETIME* ft)
+static UINT64 FileTimeTo100ns(const FILETIME* ft)
```

Comment:
This is the only change in here, correct?
Better remove that as it is a change not related to your tcp patch.

Reply:
yes
removed and tested, everything is ok